Utils/AutoLoad.py

```diff
@@ -0,0 +1,8 @@
+import inspect
+
+
```

Comment:
I don't think this file is necessary. All you need is to pass on to the class the config file. Why the new function?